### ISSUE TITLE
Add OPDB OpenType feature to all fonts

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#375]
Most Latin fonts don't include a OPBD OpenType feature, which means they are left aligned by bounding box, so when you have a title with a left flat side on the first glyph, it appears indented. Eg,

> # Banana
> 
> wrap

So, lets write a script to add a OPBD feature to all the fonts that solves this.

**Comment by simoncozens** [comment]
This might be a good idea - but it's out of scope for fontbakery!

**Comment by behdad** [comment]
> @behdad could you shed more light on this? :)

FWIW HarfBuzz now has API for this (`hb_ot_layout_lookup_get_optical_bound`). But no client uses it as far as I know. Might be wrong. @matthiasclasen

**Comment by kenmcd** [comment]
Are there any example fonts which have `opbd`?
I have only found examples of `lfbd` and `rtbd`.
EB Garamond 12 (2014) has `lfbd` and `rtbd`.
Linux Libertine (incl. with LibreOffice) has `lfbd` and `rtbd`.
Catalyst (commercial) has `lfbd` and `rtbd`.
And I have tested these.

LibreOffice supports `lfbd` and `rtbd`.
Have not tested `opbd` (no font), but I assume if HarfBuzz supports it that it is supported.
Update: Tested `opbd` with Adam Twardoch's Nadyezhda SL One font (for testing OpenType features support) - and it shows it is supported in LibreOffice

Affinity Publisher supports `lfbd` and `rtbd`, and not `opbd`.
Their comment was basically why bother with `opbd` if you have `lfbd` and `rtbd`.

So at least two fairly prominent applications support it.
